sites/partners/styles/overrides.scss

```diff
@@ -2,4 +2,14 @@
   .field-value-error {
     --field-value-label-color: var(--seeds-color-red-500);
   }
+
+  .seeds-tag {
+    text-transform: uppercase;
+    letter-spacing: var(--seeds-letter-spacing-wider);
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+  }
```

Comment:
This makes sense to me that these overrides are needed in order to make the tag look correct. However, I'm a bit surprised how much of it is custom css and not token overriding. Especially since you are overriding it for all tags on all of partners

Reply:
https://github.com/bloom-housing/ui-seeds/blob/main/src/text/Tag.scss The existing tokens are more for styling of the tag itself which is expected, these styles here are mostly for how it fills the container. The text transform and letter spacing are technically not styles that I think we would recommend anyone to change / we are eventually moving away from uppercasing text, but this gets us to parity.

Reply:
The letter spacing makes sense as well as the uppercase. I'm just wondering if we will run into issues down the line with these overrides being on the root level of partners. 
For example in the future we might want a tag that is not 100% width. I guess for now this is probably ok to keep here until we have that use case come up.